From 98ccbfe99d7c0d56e2550f24e998cb43aaff0365 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonah=20Br=C3=BCchert?= <jbb@kaidan.im>
Date: Sat, 16 Jul 2022 19:30:29 +0200
Subject: [PATCH] ytmusic: Include iostream

Previously this was not necessary, but now it fails to compile.
I suspect this to be related to a gcc update.
---
 src/ytmusic.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ytmusic.cpp b/src/ytmusic.cpp
index 1c338a5..2ccd1f3 100644
--- a/src/ytmusic.cpp
+++ b/src/ytmusic.cpp
@@ -5,6 +5,7 @@
 #include "ytmusic.h"
 
 #include <algorithm>
+#include <iostream>
 
 #include <pybind11/embed.h>
 #include <pybind11/stl.h>
-- 
2.35.1

